pkg/agent/route/route_linux.go

```diff
@@ -130,6 +131,11 @@ func (c *Client) Initialize(nodeConfig *config.NodeConfig, done func()) error {
 		return fmt.Errorf("failed to initialize ip routes: %v", err)
 	}
 
+	// Ensure IPv6 forwarding is enabled if it is a dual-stack or IPv6-only cluster.
+	if c.nodeConfig.NodeIPv6Addr != nil && sysctl.EnsureSysctlNetValue("ipv6/conf/all/forwarding", 1) != nil {
+		return fmt.Errorf("failed to enable IPv6 forwarding")
+	}
```

Comment:
should this be the responsibility of the agent? does it require privileged mode (I believe we should avoid new operations that require privileged mode - see https://github.com/antrea-io/antrea/issues/1363). If this parameter needs to be enabled, maybe we should check the value and fail if it's not set. Alternatively, we can check & set it from a privileged initContainer...

Reply:
I am bit confused. You mean controller should take care of this?

Used `sysctl.GetSysctlNet` to check the value.

Reply:
I didn't mention the controller. It should either be the responsibility of the user (in which case we just have a check and fail if IPv6 forwarding is disabled) or we should do it in a separate privileged initContainer. I prefer the former. If you want to use IPv6, it's reasonable to expect that you enable IPv6 on your Nodes...

Reply:
Got it. I put a check there.